priv/static/app.css

```
@@ -2530,6 +2530,342 @@ textarea {
   min-height: 100px;
   resize: vertical;
 }
+.pricing-panel {
+  margin-bottom: 16px;
+  padding: 20px;
+  border: 1px solid var(--border);
+  @supports (color: color-mix(in lab, red, red)) {
+    border: 1px solid color-mix(in srgb, var(--border) 96%, transparent);
+  }
+  border-radius: 10px;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)